jvm/src/main/scala/eval/ParallelInterpreter.scala

```diff
@@ -0,0 +1,65 @@
+package com.azavea.maml.eval
+
+import com.azavea.maml.ast._
+import com.azavea.maml.error._
+import com.azavea.maml.eval.directive._
+
+import cats._
+import cats.implicits._
+import cats.data.Validated._
+import cats.data.{NonEmptyList => NEL, _}
+import cats.effect.ContextShift
+
+import scala.reflect.ClassTag
+
+class ParallelInterpreter[F[_]: Monad, G[_]](directives: List[Directive])(
```

Comment:
`F[_]` has to be a Monad for `parTraverse` reasons -- I asked for weaker evidence and the compiler got mad